api/src/main/resources/marquez/db/migration/R__1_jobs_view_and_rewrite_function.sql

```diff
@@ -0,0 +1,127 @@
+CREATE OR REPLACE VIEW jobs_view
```

Comment:
_Minor_: I would be more descriptive with the description for the repeatable migration:

> **R__1_add_jobs_view_and_rewrite_jobs_fqn_table_function.sql**

Reply:
I don't know... after the first time it runs, it's not really _adding_ anything anymore. I kind of like treating the repeatable migration file just like a code file - you'd never use `CreateJobDao.java` as the file name.
Now, I think one legitimate criticism might be whether the `rewrite_function` and the `jobs_view` ought to be in the same migration file (if we're thinking of the file like code). But, IMO, they're so intrinsically connected it made sense to keep them together.

Reply:
I think that works, I'd like to document how we define migrations (or repeatable ones) for new contributors. So, as long as we have a consistent naming convention, and it's documented then we can leave the name of the file as is (and looks like `R__People_view.sql` is used in Flyways docs on [repeatable migrations](https://flywaydb.org/documentation/tutorials/repeatable#creating-a-repeatable-migration), which is more aligned with your naming approach)

Reply:
Ok, I'll update the files to follow that naming convention and add a line to the `CONTRIBUTING.md` file